test/manifests/multiperiod-overlapped.mpd

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" xmlns:cenc="urn:mpeg:cenc:2013" profiles="urn:mpeg:dash:profile:isoff-live:2011" type="static" mediaPresentationDuration="PT36.269S" minBufferTime="PT2.000S">
+  <Period id="0" duration="PT5.972633333S">
+    <AdaptationSet mimeType="audio/mp4" segmentAlignment="true" startWithSAP="1" lang="en">
+      <Role schemeIdUri="urn:mpeg:dash:role:2011" value="main"></Role>
+      <SegmentTemplate initialization="https://example.com/$RepresentationID$/init0.m4f" media="https://example.com/$RepresentationID$/segment$Number$.m4f" startNumber="0" timescale="44100">
+        <SegmentTimeline>
+          <S d="88064" r="3"></S>
+        </SegmentTimeline>
+      </SegmentTemplate>
+      <Representation audioSamplingRate="44100" bandwidth="93000" codecs="mp4a.40.2" id="default_audio96_2"></Representation>
+    </AdaptationSet>
+    <AdaptationSet mimeType="video/mp4" scanType="progressive" segmentAlignment="true" startWithSAP="1">
+      <Role schemeIdUri="urn:mpeg:dash:role:2011" value="main"></Role>
+      <SegmentTemplate initialization="https://example.com/$RepresentationID$/init0.m4f" media="https://example.com/$RepresentationID$/segment$Number$.m4f" startNumber="0" timescale="30000">
+        <SegmentTimeline>
+          <S d="60060" r="2"></S>
+          <S d="59059"></S>
+        </SegmentTimeline>
+      </SegmentTemplate>
+      <Representation bandwidth="884000" codecs="avc1.4d001e" frameRate="2997/100" height="360" id="default_video900_1_640x360" width="640"></Representation>
+    </AdaptationSet>
+  </Period>
+  <Period id="1" duration="PT4.838166665S">
```

Comment:
There is no harm adding comments in the xml to explain how the 2 periods are overlapping...
as far as I remember, if a start time of a period is not specified, its startime will be the start of previous one + the duration of it...
So how the periods are overlapping? is it due to the segment timelines?...
Please comment the file, as this is very not straight forward to understand / remember and why such test input should end up with the expected parsed playlist...